SCAutolib/cli_commands.py

```diff
@@ -165,7 +169,47 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
+@click.command()
+@click.option('--init',
+              required=False,
+              default=False,
+              is_flag=True,
+              help="Initialize GUI for manual testing")
+@click.option("--assert-text",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Check if a word is found on the screen ")
+@click.option("--click-on",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Click on object containing word")
+@click.option("--check-home-screen",
+              required=False,
+              is_flag=True,
+              default=False,
+              show_default=True,
+              help="Check if screen appears to be the home screen")
+@click.pass_context
+def gui(ctx, init, assert_text, click_on, check_home_screen):
+    """ Run GUI actions manually """
+    from SCAutolib.models.gui import GUI
+    gui = GUI()
+    if init:
+        gui.__enter__()
+    elif assert_text is not None:
```

Comment:
Why elif? Don't we want to have a command line "scauto ... gui --init --click-on test --check-home-screen"?

Reply:
What I was thinking (I don't know if it is possible yet) is to have a list of command objects that will be specified by the arguments and then run the items in that list.

Reply:
Ok, I've refactored the code and updated some things that make this run much better now I think.   Can you take a look at the recent updates?